bio.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="bio.css" />
+    <title>Portfolio Bio Page</title>
+  </head>
+  <body>
+    <header>
+      <nav class="header-nav">
+        <a href="home.html">Home</a>
+        <a href="bio.html">Bio</a>
+        <a href="experience.html">Experience</a>
+        <a href="portfolio.html">Portfolio</a>
+      </nav>
+    </header>
+
+    <main>
+      <h1>About Me</h1>
+
+      <div class="picture-div">
+        <img class="display-pic" alt="bitmoji picture" src="bio.png" />
+      </div>
+
+      <div class="p-div">
+        <p>
+          Hello, I'm Andrew! I am a full-stack Engineer based in New York City.
+          After studying the hardware side of tech as an Electrical Engineer
+          student, I decided to switch from a breadboard to a keyboard and learn
+          how to build things on the web. Since middle school, I have developed
+          a passion for collecting sneakers, while my lifelong fascination with
+          cars centers on Japanese and European models. I am also a NY sports
+          fan, specifically of the Knicks, Mets, and Giants.
+        </p>
+      </div>
+    </main>
+
+    <footer>
```

Comment:
nice work with a consistent header and footer on each page - helps with a consistent experience across the site